regression/pages/lms/course_page_lms.py

```diff
@@ -22,3 +22,13 @@ def click_resume_button(self):
         Clicks Resume button of the course selected
         """
         self.q(css='.last-accessed-link').first.click()
+
+    def get_pages_name(self):
```

Comment:
Ok, I think this is moreso giving you the list of tabs. Should probably clarify that it's tabs and not individual pages (which appear in a sequence). Please rename the method?